app/utils/mixpanel.js

```diff
@@ -0,0 +1,14 @@
+import ENV from 'website-my/config/environment';
+import mixpanel from 'mixpanel-browser';
+
+mixpanel.init(ENV.mixpanel, { debug: true });
```

Comment:
It is mentioned as `ENV.MIXPANEL_TOKEN` in the environment file. What are you trying to access by `ENV.mixpanel`?

Reply:
Ok, I guess I pushed the code for things I was trying out, but it also didn't work with that `ENV.MIXPANEL_TOKEN ` also

Reply:
Did you have the MIXPANEL_TOKEN variable exported in your terminal before starting the server?

Reply:
https://askubuntu.com/questions/58814/how-do-i-add-environment-variables